Sources/RichTextKit/Component/RichTextViewComponent+Link.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  RichTextViewComponent+Link.swift
+//
+//
+//  Created by Dominik Bucher on 04.12.2023.
+//
+
+import Foundation
+import UIKit
+
+public extension RichTextViewComponent {
+    /// Unsets the current link at the given range.
+    /// This method takes the range, checks if there are any attributes for link and removes them.
+    /// After that, sets default typing attributes so we can continue writing as is.
+    /// Note if nothing is selected, we just set typing attributes so we can continue writing in normal pace
+    func unsetLinkFromCurrentRichTextStyle() {
+        if hasSelectedRange {
+            richText.enumerateAttributes(in: selectedRange) { attributes, range, _ in
+                let mutableAttributedString = NSMutableAttributedString(attributedString: richText)
+                attributes.forEach { attribute, _ in
+                    mutableAttributedString.removeAttribute(attribute, range: range)
+                    mutableAttributedString.fixAttributes(in: range)
+                    mutableAttributedString.addAttribute(.font, value: FontRepresentable.standardRichTextFont, range: range)
+                    mutableAttributedString.addAttribute(.foregroundColor, value: UIColor.label, range: range)
+                }
+                setRichText(mutableAttributedString)
+            }
+        }
+        
+        self.typingAttributes = [.font: FontRepresentable.standardRichTextFont, .foregroundColor: UIColor.label]
```

Comment:
This is probably fragile... I need to figure out how to correctly set the typing attributes. Any idea @danielsaidi ? :)

Reply:
Can you use the already existing font and color functions?